tests/manage/ocs_288.py

```diff
@@ -0,0 +1,436 @@
+    "templates/CSI/cephfs", "secret.yaml"
+)
+TEMP_YAML_FILE = 'temp.yaml'
+TEMP_YAML_FILE_FS = 'temp_fs.yaml'
+
+SC = ocp.OCP(
+    kind='StorageClass', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+POOL = ocp.OCP(
+    kind='CephBlockPool', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+POD = ocp.OCP(
+    kind='Pod', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+SECRET = ocp.OCP(
+    kind='Secret', namespace="default"
+)
+PVC = ocp.OCP(
+    kind='PersistentVolumeClaim', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+CFS = ocp.OCP(
+    kind='CephFilesystem', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+
+
+@pytest.fixture(scope='class')
+def test_fixture(request):
+
+    def finalizer():
+        teardown()
+
+    request.addfinalizer(finalizer)
+    setup()
+
+
+def setup():
+    """
+    Setting up the environment for the test
+    """
+    assert create_secret(file_name=SECRET_RBD)
+    assert create_secret(file_name=SECRET_CEPHFS)
+    assert create_cephfilesystem()
+    assert create_multiple_rbd_storageclasses(count=5)
+    assert create_storageclass_cephfs()
+
+
+def teardown():
+    """
+    Tearing down the environment
+    """
+    assert delete_pvc()
+    assert delete_cephblockpool()
+    assert delete_storageclass()
+    assert delete_cephfilesystem(file_name=TEMP_YAML_FILE_FS)
+    assert delete_secret()
+
+
+def delete_pvc():
+    """
+    Function to delete pvc
+    :param pvc_list:
+    :return:
+    """
+    pvc_list = get_pvc()
+    for item in pvc_list:
+        log.info(f"Deleting pvc {item}")
+        assert PVC.delete(resource_name=item)
+    return True
+
+
+def delete_storageclass():
+    """
+    Function for deleting Storageclass and CephBlockPool
+    :param storageclass_list:
+    :return:
+    """
+    storageclass_list = get_storageclass()
+    for item in storageclass_list:
+        log.info(f"Deleting StorageClass with name {item}")
+        assert SC.delete(resource_name=item)
+    return True
+
+
+def delete_cephblockpool():
+    pool_list = get_cephblockpool()
+    for item in pool_list:
+        log.info(f"Deleting CephBlockPool with name {item}")
+        assert POOL.delete(resource_name=item)
+    return True
+
+
+def delete_cephfilesystem(file_name):
+    """
+    Function to delete CephFileSystem
+    :param file_name:
+    :return:
+    """
+    log.info("Deleting CephFileSystem")
+    assert CFS.delete(yaml_file=file_name)
+    return True
+
+
+def delete_secret():
+    """
+    Function to delete Secret
+    :param secret_list:
+    :return:
+    """
+    secret_list = get_secret()
+    for item in secret_list:
+        log.info(f"Deleting secret {item}")
+        assert SECRET.delete(resource_name=item)
+    return True
+
+
+def create_rbd_pool(pool_name):
+    """
+    Create Blockpool with specified name
+
+    :param pool_name:
+    :return:
+    """
+    pool_data = {}
+    pool_data['rbd_pool'] = pool_name
+    file_y = templating.generate_yaml_from_jinja2_template_with_data(
```

Comment:
Templating method is gone,  Can you check the new OCS() class and use that instead. If you need help ping me in QE chat.

Reply:
Done